css/layout.css

```diff
@@ -0,0 +1,91 @@
+.banner {
+  margin-top: 20px;
+  display:flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-content: center;
+  align-items: baseline;
+  margin-bottom: 35px;
+}
+
+.banner__logo {
+  display:flex;
+  flex-direction: row;
+  flex-wrap: no-wrap;
+  justify-content: flex-start;
+  align-content: flex-start;
+  align-items: flex-end;
+}
+
+.banner__logo h1 {
+  margin-bottom: -10px;
+}
+
+.banner__logo h2 {
+  margin-top: -5px;
+  margin-left: 10px;
+}
+
+.banner__logo img {
+  align-self: center;
+  flex-shrink: 0;
+  margin-top: 40px;
+}
+
+.recipe__header {
+  display:flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-content: center;
+  align-items: center;
+}
+
+.recipe__header h1 {
+  margin-bottom: 0px;
+}
+
+.recipe__header__print {
+  margin-bottom: -35px;
+}
+
+.recipe__ingredients {
+    display:flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    align-content: center;
+    align-items: center;
```

Comment:
This basic flex parameter seems to be repeated several times for all the different sections. Why not DRY out your code and just separate it into its own class name like .flex-wrap-center and apply it to the relevant elements in your HTML?